### PR TITLE
Fix 5-1 test suites

### PR DESCRIPTION
**Pull request by sunilkumarn417**
1) Update ceph-stable-release in upgrade test suites.
2) Added new 5.1 test suites where NFS is deployed.
3) Added exit status check for cephadm-adopt playbook execution.
4) Remove grafana deployment with beta image.

Signed-off-by: sunilkumarn417 <sunnagar@redhat.com>

### CODE REVIEW

File: pipeline/scripts/5/1/tier-0/test-rhceph-deployment-features.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ random_string=$(cat /dev/urandom | tr -cd 'a-z0-9' | head -c 5)
 instance_name="ci-${random_string}"
 platform="rhel-8"
 rhbuild="5.1"
-test_suite="suites/pacific/cephadm/tier_0_cephadm.yaml"
+test_suite="suites/pacific/cephadm/tier0_5_1_cephadm.yaml"
```
Comment (psathyan) [inline pipeline/scripts/5/1/tier-0/test-rhceph-deployment-features.sh:8]: Can we keep the file naming consistent? For example: `tier_0_cephadm_5.1.yaml`

File: pipeline/scripts/5/1/tier-1/test-cephadm-apply-service-spec.sh [hunk 2]
```diff
@@ -5,7 +5,7 @@ random_string=$(cat /dev/urandom | tr -cd 'a-z0-9' | head -c 5)
 instance_name="ci-${random_string}"
 platform="rhel-8"
 rhbuild="5.1"
-test_suite="suites/pacific/cephadm/tier1_service_apply_spec.yaml"
+test_suite="suites/pacific/cephadm/tier1_5_1_service_apply_spec.yaml"
```
Comment (psathyan) [inline pipeline/scripts/5/1/tier-1/test-cephadm-apply-service-spec.sh:8]: Let's have the file name as `tier_1_service_apply_spec_5.1.yaml`

File: suites/pacific/cephadm/tier0_5_1_cephadm.yaml [hunk 3]
```diff
@@ -0,0 +1,728 @@
+tests:
```
Comment (psathyan) [inline suites/pacific/cephadm/tier0_5_1_cephadm.yaml:1]: IMO, let's have the file name as `tier_0_cephadm_5.1.yaml`

File: suites/pacific/cephadm/tier0_5_1_cephadm.yaml [hunk 4]
```diff
@@ -0,0 +1,728 @@
...
+  - test:
```
Comment (psathyan) [inline suites/pacific/cephadm/tier0_5_1_cephadm.yaml:2]: Can we have a docstring on why we need a different file for RHCS 5.1?

File: suites/pacific/cephadm/tier1_5_1_service_apply_spec.yaml [hunk 5]
```diff
@@ -0,0 +1,242 @@
+#===============================================================================================
```
Comment (psathyan) [inline suites/pacific/cephadm/tier1_5_1_service_apply_spec.yaml:1]: Let's rename the file to `tier_1_service_apply_spec_5.1.yaml`

**Review by psathyan — COMMENTED** [review]
Superficial changes... I believe we should start having a guideline on the file name format.

**Comment by sunilkumarn417** [comment]
> Superficial changes... I believe we should start having a guideline on the file name format.

The Ceph changes(Mainly with cephadm orch deployment options) we notice is occurring between Z and minor releases, I think documenting the things at test suite level will help us.

File: pipeline/scripts/5/1/tier-0/test-rhceph-deployment-features.sh (hunk 1 above)
Comment (sunilkumarn417) [reply pipeline/scripts/5/1/tier-0/test-rhceph-deployment-features.sh:8]: As per our discussion, separated out build-version values against test suite name.

File: pipeline/scripts/5/1/tier-1/test-cephadm-apply-service-spec.sh (hunk 2 above)
Comment (sunilkumarn417) [reply pipeline/scripts/5/1/tier-1/test-cephadm-apply-service-spec.sh:8]: As per our discussion, separated out build-version values against test suite name.

File: suites/pacific/cephadm/tier1_5_1_service_apply_spec.yaml (hunk 5 above)
Comment (sunilkumarn417) [reply suites/pacific/cephadm/tier1_5_1_service_apply_spec.yaml:1]: As per our discussion, separated out build-version values against test suite name.

File: suites/pacific/cephadm/tier0_5_1_cephadm.yaml (hunk 3 above)
Comment (sunilkumarn417) [reply suites/pacific/cephadm/tier0_5_1_cephadm.yaml:18]: As per our discussion, separated out build-version values against test suite name.

**Review by psathyan — APPROVED** [review]

**Pull request merged**